### PR TITLE
ref(encoding): Promise<void> return types are unnecessary boilerplate.

### PR DESCRIPTION
**Pull request by getspooky**
We should strive for concise code. `void` and `Promoise<void>` return types are unnecessary boilerplate.
Discussion https://github.com/denoland/deno/issues/8832

### CODE REVIEW

File: encoding/csv_test.ts [hunk 1]
```diff
@@ -473,16 +473,13 @@ for (const t of testCases) {
...
-            new BufReader(new StringReader(t.Input ?? "")),
-            {
-              separator,
-              comment: comment,
-              trimLeadingSpace: trim,
-              fieldsPerRecord: fieldsPerRec,
-              lazyQuotes: lazyquote,
-            },
-          );
+          await readMatrix(new BufReader(new StringReader(t.Input ?? "")), {
```
Comment (wperron) [inline encoding/csv_test.ts:476]: We don't need the formatting change, you can revert this

**Review by wperron — CHANGES_REQUESTED** [review]
Thanks, just a small comment on the formatting change.

File: encoding/csv_test.ts (hunk 1 above)
Comment (getspooky) [reply encoding/csv_test.ts:476]: @wperron fixed

Comment (wperron) [reply encoding/csv_test.ts:476]: Thanks, but I'm really just interested in the changes on line 452 and 624

Comment (getspooky) [reply encoding/csv_test.ts:476]: @wperron done

**Review by wperron — APPROVED** [review]
LGTM, thanks

**Pull request merged**